pom.xml

```diff
@@ -47,10 +47,11 @@
         <kafka.embedded.env.version>3.2.1</kafka.embedded.env.version>
         <orgjson.version>20220924</orgjson.version>
         <junit.version>5.9.1</junit.version>
+        <gson.version>2.10</gson.version>
 
         <!-- Interne versjoner -->
         <unleash.version>6.1.0</unleash.version>
-        <oauth2-oidc-sdk.version>9.43.1</oauth2-oidc-sdk.version>
+        <oauth2-oidc-sdk.version>10.1</oauth2-oidc-sdk.version>
```

Comment:
Det er ikke noe braking changes changes?

Reply:
Finner ikke release notes... men testerne kjører jo fint :D

Reply:
* Breaking change, upgrades OpenID Connect Federation 1.0 support to draft
      24.

token-support 2.1.7 er over på 10.0 med så jag tipper det bare er o merge & adapt :)

Reply:
Ok, det er jo bra. 
Jeg stoler uansett ikke helt på testdekningen i denne appen :D